.github/workflows/create-botbuilder-v5-parity-issue.yml

```diff
@@ -0,0 +1,68 @@
+name: create-botbuilder-v5-parity-issue.yml
+
+on:
+  pull_request:
+    branches:
+    - main
+    types: [closed]
+
+jobs:
+  encode:
+    name: encode inputs
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+
+    outputs:
+      pr-body: ${{ steps.pr-body.outputs.value }}
+      pr-number: ${{ steps.pr-number.outputs.value }}
+      pr-title: ${{ steps.pr-title.outputs.value }}
+      repository: ${{ steps.repository.outputs.value }}
+
+    steps:
+      - id: pr-body
+        uses: joshgummersall/base64-encode@main
+        with:
+          value: ${{ github.event.pull_request.body }}
+
+      - id: pr-number
+        uses: joshgummersall/base64-encode@main
```

Comment:
Should we have a local copy?

Reply:
Please feel free to fork - the code is pretty minimal: https://github.com/joshgummersall/base64-encode